compute/createVM/go.mod

```
@@ -1,4 +1,4 @@
-module azure-sdk-for-go-samples/compute
+module azure-sdk-for-go-samples/createVM/compute
```

I believe this should be 
```suggestion
module azure-sdk-for-go-samples/compute/createVM
```
in order to align the directory hierarchy